js/js.dce/src/org/jetbrains/kotlin/js/dce/Context.kt

```diff
@@ -180,7 +216,23 @@ class Context {
             }
             private set
 
-        val members: Map<String, Node> get() = original.membersImpl
+        val members: Map<String, Node> get() = original._membersImpl ?: emptyMap()
+
+        fun addDependency(node: Node) {
+            dependenciesImpl += node
+        }
+
+        fun addFunction(function: JsFunction) {
+            functionsImpl += function
+        }
```

Comment:
This API makes it simple to add `function` to this Node's collection, instead of `this.original.functionsImpl`

Reply:
Changed function to add node to `original.functionsImpl`